core/formats/sparse_bitmap.cpp

```diff
@@ -379,7 +379,13 @@ struct container_iterator<BT_DENSE, true> {
         self->in_->seek(pos);
       } else {
         for (auto* prev_word = ctx.u64data - 1; !word; --word_idx) {
-          word = *--prev_word;
+          --prev_word;
+          if constexpr (AT_DIRECT_ALIGNED == Access) {
+            word = *prev_word;
```

Comment:
I interpret `AT_DIRECT_ALIGNED` as "we can read the memory directly, and we are 100% sure that the address we read from is correctly aligned".
If this is correct, can we add an assertion, e.g. `assert(reinterpret_cast<uintptr_t>(&prev_word) % sizeof(word) == 0);`?